alvr/server_core/src/tracking/face.rs

```diff
@@ -46,6 +46,7 @@ impl FaceTrackingSink {
             .ok();
     }
 
+    #[expect(clippy::trivially_copy_pass_by_ref)]
```

Comment:
Maybe let's do what clippy says and pass prefix by value

Reply:
I mean yea you just have to deref the prefix that's why I decided to slap on the expect instead but sure